src/client/sandbox/node/live-node-list/wrapper-base.js

```diff
@@ -0,0 +1,36 @@
+const DEFINED_PROPERTIES_COUNT = 10000;
+
+export class LiveNodeListWrapperBase {
+    constructor () {
+        const defineProperty = function (index, isEnumerable) {
+            Object.defineProperty(this, index, {
+                enumerable:   isEnumerable,
+                configurable: true,
+
+                get: function () {
+                    this._refreshNodeList();
+
+                    return this._filteredNodeList[index];
+                }
+            });
+        };
+
+        for (let i = 0; i < DEFINED_PROPERTIES_COUNT; i++)
+            defineProperty.call(this, i, false);
+
+        Object.defineProperty(this, '_defineProperty', { value: defineProperty });
+        Object.defineProperty(this, '_refreshNodeList', {
+            value: () => {
+                throw new Error('Not implemented');
+            }
+        });
+    }
+}
+
+LiveNodeListWrapperBase.prototype = NodeList.prototype;
+
+export const NODE_LIST_WRAPPER = new LiveNodeListWrapperBase();
```

Comment:
usually we use UPPER_CASE for string/numeric consts or objects with srting/numeric consts. But here is just an instance of the class. I expect to see here camelcase as usually